components/places/src/ffi.rs

```diff
@@ -255,6 +301,22 @@ impl PlacesConnection {
             Ok(())
         })
     }
+
+    fn wipe_local(&self) -> Result<()> {
```

Comment:
this is only doing history and other misc stuff (ie, not bookmarks) so even though this problem kinda before, maybe rename it to, say `wipe_local_history`? In the kotlin and swift, keep the "public" names as they were, even though they remain misleading, but add a comment saying everything other than bookmarks is removed?